lark/tools/standalone.py

```diff
@@ -185,9 +180,6 @@ def main():
         parser.print_help(sys.stderr)
         sys.exit(1)
     ns = parser.parse_args()
-    if ns.old_start is not None:
-        warn('The syntax `python -m lark.tools.standalone <grammar-file> <start>` is deprecated. Use the -s option')
```

Comment:
This should als be reflected in the parser itself.

Reply:
How do you mean?

Reply:
A few lines above this, there is a call to `parser.add_arguments('old_start')`